plenum/common/func_utils.py

```diff
@@ -0,0 +1,28 @@
+
+
+def flatten(some_list):
```

Comment:
I used it earlier, but then changed implementation. Yes, looks like it is the same. I'll remove it